src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/DraftAdjudicationDto.kt

```diff
@@ -30,6 +32,13 @@ data class IncidentDetailsDto(
   val handoverDeadline: LocalDateTime,
 )
 
+@ApiModel(value = "Incident Role")
+data class IncidentRoleDto(
+  @ApiModelProperty(value = "The incident role code", notes = "If not set then it is assumed they committed the offence on their own", example = "25a")
+  val roleCode: String?,
```

Comment:
Will be null when the offender did it all by themselves

Reply:
So does this mean that the IncidentRoleDto is effectively empty if the prisoner committed it on their own?

Reply:
Am I also right in thinking that its one adjudication for each incident role - I think that's correct